src/ports/postgres/modules/utilities/validate_args.py_in

```diff
@@ -513,11 +513,12 @@ def array_col_has_same_dimension(tbl, col):
 # ------------------------------------------------------------------------
 
 
-def explicit_bool_to_text(tbl, cols, schema_madlib):
+def explicit_bool_to_text(tbl, cols, schema_madlib, is_forced=False):
```

Comment:
I'm not clear on the need for `is_forced`. Are there platforms that have a bool-to-text cast require this patching?

Reply:
w/ @ArvindSridhar On platforms that has bool to text casting (gpdb5, pg 9.6, pg 10), we still need this to make sure we can create an array of bool and text types.

Reply:
That's a good point-we added this arg to make our dev-check pass, because our dev check didn't have an explicit ::TEXT cast and yet it was running on PG10, which supports bool to text casting. Would removing the dev check test and the is_forced arg be the better way to go?